src/context/PassageDetailContext.tsx

```diff
@@ -168,7 +168,7 @@ const initState = {
   stepComplete: (stepId: string) => {
     return false;
   },
-  discussionSize: { width: 450, height: 700 },
+  discussionSize: { width: 450, height: 900 },
```

Comment:
Changing these height values is probably a different issue from the discussion card and the user name in the export message.

Reply:
The height change is related to adding the comment to the box.  It is needed for this commit